fleet/fleet.go

```diff
@@ -74,6 +75,93 @@ type UnitStatus struct {
 
 	// Name represents the unit file name.
 	Name string
+
+	// Slice represents the slice expression. E.g. @1, or @foo, or @5., etc..
+	Slice string
+}
+
+type UnitStatusList []UnitStatus
+
+func (usl UnitStatusList) Group() ([]UnitStatus, error) {
```

Comment:
this feels like business logic that belongs to the controller, not the way we talk to fleet. It's part of the higher level stuff that does not belong in here IMHO.